src/main/java/at/hannibal2/skyhanni/features/commands/OpenLastStorage.kt

```diff
@@ -0,0 +1,125 @@
+package at.hannibal2.skyhanni.features.commands
+
+import at.hannibal2.skyhanni.events.MessageSendToServerEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.ChatUtils.senderIsSkyhanni
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object OpenLastStorage {
+    private enum class StorageType {
+        ENDER_CHEST, BACKPACK
+    }
+
+    private var lastBackpack = -1
+    private var lastEnderChest = -1
```

Comment:
please use kotlin's null instead of -1 as default/invalid vlaue

Reply:
Done